src/graphing.py

```diff
@@ -75,11 +75,11 @@ def sliderChange(self):
 
     def updateToFitExp(self): #Hill climbing functions used and cllaed here
         self.updateGraph(self.m.adjustScoresToExemplary())
-        self.slider.setValue(self.m.adjustScoresToExemplary())
+        self.slider.setValue(int(self.m.adjustScoresToExemplary()))
```

Comment:
Nicely spotted - @grumpman 's change to +-2% could potentially have caused issues for storing floats in the db instead of ints. This catches it nicely.